### PR TITLE
Chaos alliance test small refactor

### PR DESCRIPTION
**Pull request by GalacticChimp**
Some small cleanups and refactors. Sadly I didn't do much because most of the issues are caused by outside code:
- no separate Chapter obj
- unclear approach do diplomacy/dialogue
once we refactor them, we can return to this script

### CODE REVIEW

File: scripts/scr_dialogue/scr_dialogue.gml
Comment (GalacticChimp) [inline scripts/scr_dialogue/scr_dialogue.gml:1]: only renamed argument0 to diplo_keyphrase, to make things clear in code

File: scripts/scr_diplomacy_chaos_accept_peace/scr_diplomacy_chaos_accept_peace.gml
Comment (GalacticChimp) [inline scripts/scr_diplomacy_chaos_accept_peace/scr_diplomacy_chaos_accept_peace.gml:1]: mostly renamed vars to make the code easier to read, added some comments and simplified some conditional expressions

**Comment by OH296** [comment]
@GalacticChimp I've been doing a load of work in the diplomact and dialogue area and have some thoughts on how to simplify. a lot of confusion as all diplomacy/dialogue click events are handled in the global click event which has just become bloated. I'll have a look at the chaos alliance area do some tweaking to go in with the current bit on warp democracy i'm working on

**Comment by ZAB909** [comment]
ngl I don't see a reason to have this merged if things are gonna get redone, ya know? thanks though man

**Pull request closed without merging**